vault/logical_system_paths.go

```diff
@@ -453,15 +612,84 @@ func (b *SystemBackend) rekeyPaths() []*framework.Path {
 				},
 			},
 
+			// TODO no callbacks here as well
```

Comment:
Have you been able to figure out where the callbacks are or which functions get called?

Reply:
Yes I did and updated them just forgot to remove this one... lol